go-modules-update.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -x
+
+die () {
+    echo >&2 "\nERROR: $@\n"
+    exit 1
+}
+
+while read repo branches; do
+    if [ "$repo" != "#" ]; then
+    (
+        cd $repo || die "$repo: does not exit"
+        git fetch origin || die "$repo: git fetch"
+        for i in $branches; do
+            git checkout -B module-update-$i origin/$i || die "$repo:$i checkout"
+            rm -rf .git/MERGE*
+            if ! git subtree pull --squash --prefix=release-tools https://github.com/kubernetes-csi/csi-release-tools.git master; then
```

Comment:
This script is also updating csi-release-tools. Do we want to include that in the script or have separate scripts?

Reply:
What do you mean separate? Do you mean everything in go-get-kubernetes?

Reply:
should we have 2 different scripts? One for updating csi-release-tools, and another for updating kubernetes dependencies?

Reply:
Hůlky